tools/deb/istio-start.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Script to run Envoy for Istio.
+
```

Comment:
The intent is for iptables script to be safe if sidecar.env is missing - and it should be possible to run it.
I'm still testing and tweaking the special cases - probably this can be improved in separate PR.